core/src/main/java/jenkins/model/GlobalCloudConfiguration.java

```diff
@@ -1,30 +1,56 @@
 package jenkins.model;
 
 import hudson.Extension;
+import hudson.RestrictedSince;
+import hudson.model.Descriptor;
+import hudson.model.RootAction;
 import hudson.slaves.Cloud;
+import hudson.util.FormApply;
 import net.sf.json.JSONObject;
 import org.jenkinsci.Symbol;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
 import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.verb.POST;
 
+import javax.annotation.CheckForNull;
+import javax.servlet.ServletException;
 import java.io.IOException;
 
 /**
- * Adds the {@link Cloud} configuration to the system config page.
+ * Provides a configuration form for {@link Jenkins#clouds}.
  *
- * <p>
- * This object just acts as a proxy to configure {@link Jenkins#clouds}
- *
- * @author Kohsuke Kawaguchi
+ * Has been overhauled in Jenkins 2.XXX to no longer contribute to Configure System, but be a standalone form.
  */
-@Extension(ordinal=-100) @Symbol("cloud") // historically this was placed at the very end of the configuration page
-public class GlobalCloudConfiguration  extends GlobalConfiguration {
+@Extension
```

Comment:
Why the ordinal before?

Reply:
The comment on the same line explains it.